src/types.rs

```diff
@@ -63,6 +63,8 @@ pub enum ColumnRef {
     Column(DynIden),
     TableColumn(DynIden, DynIden),
     SchemaTableColumn(DynIden, DynIden, DynIden),
+    Wildcard,
+    TableWildcard(DynIden)
```

Comment:
@billy1624 how will this affect SeaORM / SeaSchema?

Reply:
No impact for SeaSchema. But this part of code has to be updated in SeaORM.
https://github.com/SeaQL/sea-orm/blob/82bad4a376efb875ac32ab601ac2cd9cce191e3f/src/query/combine.rs#L45-L62